da/px-platform.json

```diff
@@ -471,6 +471,11 @@
             "title": "Beklager, vi kan ikke betale dig...",
             "description": "Din PayPal-konto er blevet frakoblet fra IntelliZoom. For at blive betalt og deltage i fremtidige undersøgelser, skal din PayPal-konto være tilkoblet.",
             "cta": "Tilknyt PayPal"
+          },
+          "notVerified": {
+            "title": "Handling nødvendig for PayPal-betalinger",
+            "description": "En tilsluttet og verificeret PayPal-konto er påkrævet for at deltage i betalte forskningsundersøgelser. Gå først til [Hvordan verificerer jeg min PayPal-konto?](https://www.paypal.com/cshelp/article/how-do-i-verify-my-paypal-account-help434) Når den er verificeret, skal du vende tilbage hertil og vælg \"Bekræft PayPal-verificering\".  ",
+            "cta": "Bekræft PayPal-bekræftelse"
```

Comment:
Hope the button still looks okay with the longer text...

Reply:
Just checked, looks ok 
<img width="326" alt="Screenshot 2024-05-23 at 14 52 05" src="https://github.com/IntelliZoomPlatform/px-translations/assets/136830723/bf21e894-afe6-45f7-bc95-ccc1ee48ce03">

Reply:
Thanks for checking! I've actually just submitted another commit because I remembered that the line spacing for PayPal error page needed an additional `\n` (got it updated on Phrase as well, I originally pushed the token with only one `\n`). And added en-GB tokens. We didn't receive translations for en-GB but since that file is there we should probably keep it up to date? 🤷‍♀️